src/Compute/Compute/help/Add-AzVhd.md

```diff
@@ -289,21 +337,6 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Zone
```

Comment:
Is parameter `-Zone` removed?

Reply:
no i changed the name to "diskzone" to make it more consistent and added "zone" as alias